pentaho-gwt-widgets/src/org/pentaho/gwt/widgets/client/colorpicker/Color.java

```diff
@@ -516,7 +516,7 @@ public int getRed() {
    * @return green component
    */
   public int getGreen() {
-    return (int) ( g * 255 );
+    return (int) ( g * 255.0f + 0.5f );
```

Comment:
Not sure I understand why this is necessary. Why add 0.5? Why transit through a float format only to loose the decimal part after the (int) cast?